test/browser.js

```diff
@@ -61,3 +61,55 @@ test('aborting a request', withPage, async (t, page) => {
 
 	await server.close();
 });
+
+test('onProgress works', withPage, async (t, page) => {
+	const server = await createTestServer();
+
+	server.get('/', (request, response) => {
+		response.end('meow');
```

Comment:
This doesn't really test streaming as it's only one chunk. I'm not sure how it splits up chunks, but we should ensure it uses at least two chunks.

Reply:
Will do.